021/main.go

```diff
@@ -0,0 +1,18 @@
+// Daily Joel #21
+// Slices and zero values.
+package main
+
+import "fmt"
+
+func main() {
+	letters := make([]string, 100)
+	letters[2] = "J"
+	letters[13] = "o"
+	letters[37] = "e"
+	letters[71] = "l"
+	letters[95] = "\n"
+
+	for _, l := range letters {
+		fmt.Print(l)
+	}
```

Comment:
We then iterate over the range of letters printing l. Very inefficient @vmorsell 👏🏻